website/content/docs/enterprise/admin-partitions.mdx

```diff
@@ -55,6 +55,8 @@ Only resources in the `default` admin partition will be replicated to secondary
 
 Client agents will be configured to operate within a specific admin partition. The DNS interface will only return results for the admin partition within the scope of the client.
 
+-> **Consul Dataplane and DNS**: As of Consul v1.14, The Consul clients are optional in Kubernetes since the introduction of [Consul Dataplane](/consul/docs/connect/dataplane), however, to utilize the Consul DNS for service discovery a Consul client is required in the client admin partitions
```

Comment:
```suggestion
By default, Consul on Kubernetes uses [Consul dataplanes](/consul/docs/connect/dataplane) instead of client agents to manage communication between service instances. But to use the Consul DNS for service discovery, you must start a Consul client in client admin partitions.
```
I don't think we need to format this as into a callout box. Also, we can lean on the docs version to indicate when behavior changed and no longer need to call out specific versions.

Reply:
@am-ak Is this for service discovery only? If so the preferred way of doing service discovery only is via Service Sync.

Reply:
Hi @david-yu, I replied in slack https://hashicorp.slack.com/archives/C011HP57U0Y/p1691660804668289